src/kirby/components/list/list-cell-line/list-cell-line.component.scss

```diff
@@ -0,0 +1,9 @@
+.primary {
+  font-size: 1.2em;
```

Comment:
Should we not use the defines sizes in the variable classes?

```scss
$sizes: (
  xxxl:         70,    // 4.375rem,         
  xxl:          60,    // 3.75rem,          
  xl:           50,    // 3.125rem,       
  l:            32,    // 2rem,           
  m:            16,    // 1rem,  - Same as gutter
  s:            12,    // .75rem,         
  xs:           8,     // .5rem, - Same as gutter-small
  xxs:          4,     // .25rem,     
  gutter:       16,    // 1rem,  - helper
  gutter-small: 8      // 0.5rem - helper
);
```

Reply:
We surely should. Now using $font-size() and $font-weigth()